scripts/spark/silver/purchased_games.py

```
@@ -47,10 +47,16 @@ def load_to_postgre_db(
     return
 
 
+def add_pratform_prefix(df: DataFrame, column: str, prefix: str):
+    df = df.withColumn(column, F.concat(F.lit(prefix), F.col(column)))
+    return df
+
+
 def main():
     username = "airflow"
     password = "airflow"
```

Ruff (0.14.7)

Possible hardcoded password assigned to: "password"

(S105)